File: pg_postgis_mvt_node/app/router.js
'use strict';

/**
 * @param {Egg.Application} app - egg application
 */
module.exports = app => {
  const { router, controller } = app;
  const namespace = "/pg_postgis_mvt/api/v1"
  router.get('/', controller.home.index);
	router.get(namespace+"/:tableName/:z/:x/:y", controller.mvt.getMvt)
};

File: pg_postgis_mvt_go/app/router/router.go
package router

import (
	"pg_postgis_mvt/app/controller"

	"github.com/kataras/iris/v12"
)

// Router iris的
func Router(irisApp *iris.Application) {
	namespace := "/pg_postgis_mvt/api/v1"
	// 为了看起来更容易理解
	router := irisApp
	router.Get(namespace+"/{tableName:string}/{z:int}/{x:int}/{y:int}", controller.Mvt.GetMvt)
}
